x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/rollup/RollupActionConfig.java

```diff
@@ -6,157 +6,206 @@
 
 /**
  * This class holds the configuration details of a {@link RollupAction} job, such as the groupings, metrics, what
  * index to rollup and where to roll them to.
+ *
+ *      * FixedInterval is a {@link RollupActionConfig} that uses a fixed time interval for rolling up data.
+ *      * The fixed time interval is one or multiples of SI units and has no calendar-awareness (e.g. doesn't account
+ *      * for leap corrections, does not have variable length months, etc).
+ *      *
+ *      * Calendar-aware interval is not currently supported
+ *
+ *  {
+ *    "fixed_interval" : "1d",
+ *    "time_zone" : "UTC"
+ *  }
  */
 public class RollupActionConfig implements NamedWriteable, ToXContentObject {
 
-    private static final String NAME = "xpack/rollup/action/config";
+    private static final String NAME = "rollup/action/config";
+    public static final String FIXED_INTERVAL = "fixed_interval";
+    public static final String TIME_ZONE = "time_zone";
+    public static final String DEFAULT_TIMEZONE = ZoneId.of("UTC").getId();
 
-    private final RollupActionGroupConfig groupConfig;
-    private final List<MetricConfig> metricsConfig;
+    private static final String timestampField = DataStreamTimestampFieldMapper.DEFAULT_PATH;
+    private final DateHistogramInterval fixedInterval;
+    private final String timeZone;
+    private final String intervalType = FIXED_INTERVAL;
 
     private static final ConstructingObjectParser<RollupActionConfig, Void> PARSER;
     static {
-        PARSER = new ConstructingObjectParser<>(NAME, false, (args) -> {
-            RollupActionGroupConfig groupConfig = (RollupActionGroupConfig) args[0];
-            @SuppressWarnings("unchecked")
-            List<MetricConfig> metricsConfig = (List<MetricConfig>) args[1];
-            return new RollupActionConfig(groupConfig, metricsConfig);
+        PARSER = new ConstructingObjectParser<>(NAME, a -> {
+            DateHistogramInterval fixedInterval = (DateHistogramInterval) a[0];
+            if (fixedInterval != null) {
+                return new RollupActionConfig(fixedInterval, (String) a[1]);
+            } else {
+                throw new IllegalArgumentException("Parameter [" + FIXED_INTERVAL + "] is required.");
+            }
         });
-        PARSER.declareObject(
-            optionalConstructorArg(),
-            (p, c) -> RollupActionGroupConfig.fromXContent(p),
-            new ParseField(RollupActionGroupConfig.NAME)
+
+        PARSER.declareField(
+            constructorArg(),
+            p -> new DateHistogramInterval(p.text()),
+            new ParseField(FIXED_INTERVAL),
+            ObjectParser.ValueType.STRING
         );
-        PARSER.declareObjectArray(optionalConstructorArg(), (p, c) -> MetricConfig.fromXContent(p), new ParseField(MetricConfig.NAME));
+        PARSER.declareStringOrNull(optionalConstructorArg(), new ParseField(TIME_ZONE));
```

Comment:
Since the team has decided that we are going to support only the UTC timezone for now, maybe we should remove the `timezone` parameter from the `_rollup` operation and add it when we add support for other timezones.

Reply:
The team decided to remove the `time_zone` parameter from the `_rollup` API and add it later more timezones are supported.

Reply:
Removed `time_zone` rollup param at [d905038](https://github.com/elastic/elasticsearch/pull/85708/commits/d905038b8eb281c2933d209af57d4bd1e9a95d20)